engine/render/renderer/platform/vulkan/utils/Queue.h

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright (c) 2022 Jonathan Moallem (@J-Mo63) & Aryeh Zinn (@Raelr)
+//
+//  This code is released under an unmodified zlib license.
+//  For conditions of distribution and use, please see:
+//      https://opensource.org/licenses/Zlib
+//
+
+#ifndef SIEGE_ENGINE_QUEUE_H
+#define SIEGE_ENGINE_QUEUE_H
+
+#include <utils/collections/StackArray.h>
+#include <volk/volk.h>
+
+#include "Types.h"
+
+namespace Siege::Vulkan::Utils
+{
+struct SubmitGraphicsCommand
+{
+    static inline constexpr size_t MAX_SEMAPHORES = 10;
+    static inline constexpr size_t MAX_COMMAND_BUFFERS = 10;
+
```

Comment:
File name does not seem to describe contents accurately?

Reply:
This is a bit more complicated to explain. The reason why this is called `Queue` is because the commands here are meant to submit draw commands to the graphics and presentation queues in the GPU. This is Vulkan-specific terminology and makes more sense in that context. I guess I could change it to `QueueCommand`, or something like that to make it clearer.